### PR TITLE
Add performance counter samples

### PR DESCRIPTION
**Pull request by kennykerr**
Just adds a basic Performance Counter API sample for both `windows` and `windows-sys`.

### CODE REVIEW

File: crates/samples/counter/src/main.rs [hunk 1]
```diff
@@ -0,0 +1,21 @@
+use windows::{core::*, Win32::System::Performance::*};
+
+fn main() {
+    unsafe {
+        let mut query = 0;
+        PdhOpenQueryW(None, 0, &mut query);
```
Comment (riverar) [inline crates/samples/counter/src/main.rs:6]:
```suggestion
        let handle = PdhOpenQueryW(None, 0, &mut query);
```

File: crates/samples/counter_sys/src/main.rs [hunk 2]
```diff
@@ -0,0 +1,21 @@
+use windows_sys::{core::*, Win32::System::Performance::*};
+
+fn main() {
+    unsafe {
+        let mut query = 0;
+        PdhOpenQueryW(std::ptr::null(), 0, &mut query);
```
Comment (riverar) [inline crates/samples/counter_sys/src/main.rs:6]:
```suggestion
        let handle = PdhOpenQueryW(std::ptr::null(), 0, &mut query);
```

File: crates/samples/counter_sys/src/main.rs [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+        loop {
+            std::thread::sleep(std::time::Duration::new(1, 0));
+            PdhCollectQueryData(query);
+
+            let mut value = std::mem::zeroed();
+            if 0 == PdhGetFormattedCounterValue(counter, PDH_FMT_DOUBLE, std::ptr::null_mut(), &mut value) {
+                println!("{:.2}", value.Anonymous.doubleValue);
+            }
+        }
+    }
```
Comment (riverar) [inline crates/samples/counter_sys/src/main.rs:20]:
```suggestion
        }
        
        PdhCloseQuery(handle);
    }
```

File: crates/samples/counter/src/main.rs [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+        loop {
+            std::thread::sleep(std::time::Duration::new(1, 0));
+            PdhCollectQueryData(query);
+
+            let mut value = Default::default();
+            if 0 == PdhGetFormattedCounterValue(counter, PDH_FMT_DOUBLE, None, &mut value) {
+                println!("{:.2}", value.Anonymous.doubleValue);
+            }
+        }
+    }
```
Comment (riverar) [inline crates/samples/counter/src/main.rs:20]:
```suggestion
        }
        
        PdhCloseQuery(handle);
    }
```

**Comment by kennykerr** [comment]
@riverar the win32 metadata really should provide native type defs for these.

**Comment by riverar** [comment]
@kennykerr Agreed, was thinking that too. Feel free to ignore the suggestions (as they'll be rendered moot when we fix that up)!

**Pull request merged**